infrasim/model.py

```diff
@@ -1782,10 +1782,15 @@ def precheck(self):
 
     @run_in_namespace
     def init(self):
-        if 'kvm_enabled' in self.__compute and not helper.check_kvm_existence():
+        if not helper.check_kvm_existence():
             self.__enable_kvm = False
+        elif 'kvm_enabled' in self.__compute and helper.check_kvm_existence():
```

Comment:
This `elif` should imply `helper.check_kvm_existence() is True`, this code should be omitted.